src/papilio/Papilio_Simulator.h

```diff
@@ -0,0 +1,864 @@
+/* -*- mode: c++ -*-
```

Comment:
This file was originally called `papilio/Simulator.h`, but due to linking object files into a static archive, this conflicted with `kaleidoscope_simulator/Simulator.h`, and I had to rename one. This is not a good name.